ui/server/src/test/scala/pl/touk/nussknacker/ui/api/ProcessesChangeListenerSpec.scala

```diff
@@ -82,9 +83,10 @@ class ProcessesChangeListenerSpec extends FunSuite with ScalatestRouteTest with
   test("listen to deployment success") {
     val processId = createProcess(processName, testCategoryName, false)
     val comment = Some("deployComment")
+    val deployComment = Some(DeploymentComment.unsafe("deployComment"))
 
-    deployProcess(processName.value, true, comment) ~> checkEventually {
-      TestProcessChangeListener.events.head should matchPattern { case OnDeployActionSuccess(`processId`, VersionId(1L), `comment`, _, ProcessActionType.Deploy) => }
+    deployProcess(processName.value, DeploySettings(validationPattern = "deployComment", ""), comment) ~> checkEventually {
```

Comment:
Hmm, I think we should check params DeploySettings are not empty strings and does validationPattern is proper regexp?

Reply:
regarding checking empty strings i'm not sure - we previously decided that empty validation pattern means comment is not required and that's also fine, result of that validation is Valid(None)

Reply:
regarding the latter, i'm not certain what we would desire proper regexps should look like, wouldn't this require regex for checking if value is regex?

Reply:
I though we can check RegExp(pattern) doesn't throw exception.. But maybe it doesn't have sens.. Just check empty strings?

Reply:
like i wrote above i'm not certain because i thought we allow empty validationPattern in DeploySettings and it means we don't require comment

Reply:
DeploymentSettings is provided as Option[DeploymentSettings], so when we don't create config then comment isn't required. On other hand when you create config then you should provide each fields: `validationPattern` and `exampleComment`.

Reply:
i understand, added validation to not allow empty if deploy config is specified, thx